cmd/snap/main.go

```diff
@@ -129,11 +129,19 @@ func init() {
 	}
 }
 
+var accessDeniedHelp = `Access denied.
+
+Please use "sudo" or the "snap login <userid>" command
```

Comment:
Thats fine with me, I'm happy to update the string. Maybe:
```Access denied.
Please use "snap login <your ubuntu sso account>".
```
(is there something nicer than `Access denied.` in the first line?